.travis.yml

```diff
@@ -323,7 +311,7 @@ matrix:
         - export TEST_EXTRA_ARGS="--unittest-ipv6"
     - <<: *macos
       osx_image: xcode11.2
-      name: xcode11, debug
+      name: xcode11.2, debug
```

Comment:
There are two builds named "xcode11.2, debug" now (here and on line 266), and I think they are identical. We only need one of these right?

Reply:
I believe on runs IPv6-specific unit tests. I don't know that really gets us much and we could remove it.